column/api/manager/credential_manager.py

```diff
@@ -0,0 +1,21 @@
+import logging
+
+from column import utils
+
+
+LOG = logging.getLogger(__name__)
+
+
+class CredentialManager(object):
+    """Column Credential Manager class
+
+    The Credential Manager layer is to support additional logic which is needed
+    to update and get credential.
+
+    """
+
+    def get_credential(self, cred):
+        return utils.get_vault_secret(cred['file_path'])
+
+    def update_credential(self, cred):
```

Comment:
Cred is payload from client, it is dict. manager would take care of extracting params from payload.